build.gradle

```diff
@@ -463,3 +463,46 @@ def configureGroovydoc(TaskProvider groovydoc) {
     include "spock/**"
   }
 }
+
+configurations {
+  kotlinCompilerClasspath {
+    canBeConsumed = false
+  }
+  kotlinScriptClasspath {
+    canBeConsumed = false
+  }
+}
+
+dependencies {
+  kotlinCompilerClasspath 'org.jetbrains.kotlin:kotlin-compiler:1.8.20'
+  kotlinCompilerClasspath 'org.jetbrains.kotlin:kotlin-scripting-compiler:1.8.20'
+  kotlinScriptClasspath('org.jetbrains.kotlin:kotlin-main-kts:1.8.20') { transitive = false }
+}
+
+def preprocessWorkflows = tasks.register('preprocessWorkflows')
```

Comment:
We should assign the tasks to a group

Reply:
Sure, no problem, which group to you suggest and to for which task?
There is `preprocessWorkflows` lifecycle task and one task for each workflow.
So just a group for the lifecycle task or groups for all tasks?
And an own group, or an existing one, and if so which?
![image](https://user-images.githubusercontent.com/325196/232437745-18c65757-255d-460a-ad36-a442fd9cc6a6.png)

Reply:
Any idea on this one?

Reply:
I'd create a `github actions` group

Reply:
Ok, added and all tasks assigned to it